.travis.yml

```diff
@@ -21,12 +21,16 @@ services:
 
 cache:
   directories:
-  - .gradle
-  - $HOME/.gradle
-  - $HOME/.m2
-  
+   - ".gradle"
+   - "$HOME/.gradle"
+   - "$HOME/.m2"
+
 notifications:
-  slack: emcnautilus:UZK3F72LSRYWXUuvlzSilFyy
+  slack:
+    matrix:
```

Comment:
Why are we changing this?

Reply:
How did you find this? It's not in the list of files I touched? Go to "Files Changed" and you can't find it.